### PR TITLE
TMA-195: LCM bricks should transfer permissions and groups

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/269/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1140/ : SUCCESS in 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/289/ : SUCCESS in 23s

File: spec/unit/actions/synchronize_user_groups_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    allow(to_project).to receive(:title)
+    allow(to_project).to receive(:uri)
+    allow(to_project).to receive(:pid) { 'to_project_id' }
+    allow(to_project).to receive(:client) { gdc_gd_client }
+  end
+
+  it 'should transfer user groups' do
+    results = subject.class.call(params)
+    expect(results).to eq [{ from: 'from_project_id', to: 'to_project_id', user_group: 'user_group_name', status: 'created' }]
+  end
```
Comment (panjan) [inline spec/unit/actions/synchronize_user_groups_spec.rb]: Please add a test for transferring the groups: `expect(GoodData::Project).to receive(:transfer_user_groups).with(...)`.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/273/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1147/ : FAILURE in 4s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/296/ : SUCCESS in 24s

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/274/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1148/ : FAILURE in 4s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/297/ : SUCCESS in 20s

**Comment by vhtien** [comment]
integration tests: https://ms-gerrit.intgdc.com/#/c/940/

**Comment by panjan** [comment]
ok to merge

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/280/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1153/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/302/ : SUCCESS in 23s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/92/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1155/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/304/ : SUCCESS in 52s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/933/ : SUCCESS in 46m 19s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/703/ : UNSTABLE in 21m 22s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/93/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1156/ : SUCCESS in 22s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/305/ : SUCCESS in 26s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/934/ : SUCCESS in 44m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/704/ : SUCCESS in 21m 17s

**Pull request merged**